x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -649,6 +649,11 @@ public enum Cap {
          */
         MATCH_FUNCTION,
 
+        /**
+         * MATCH PHRASE function
+         */
+        MATCH_PHRASE_FUNCTION(Build.current().isSnapshot()),
```

Comment:
we usually add these capabilities at the end - similar to `TransportVersions`.
I get that this results in merge conflicts many times, but it also shows a progression.
And getting them to be in chronological order helped a few times, when we had to figure out which capabilities have wrongly backported for example.
So please move this at the end - thanks!